greeneeds/projects/models.py

```diff
@@ -0,0 +1,59 @@
+from django.db    import models
+
+from users.models import User
+from core.models  import TimeStampModel
+
+class Project(TimeStampModel):
+    user            = models.ForeignKey(User, on_delete=models.CASCADE)
+    title           = models.CharField(max_length=70)
+    shortcut_title  = models.CharField(max_length=50)
+    summary         = models.CharField(max_length=100)
+    price           = models.DecimalField(max_digits=7, decimal_places=2)
+    thumbnail       = models.URLField(max_length=150)
+    pk_uri          = models.CharField(max_length=50)
+    target_amount   = models.DecimalField(max_digits=9, decimal_places=2)
+    start_datetime  = models.DateField()
+    end_datetime    = models.DateField()
+    pay_end_date    = models.DateField()
+    settlement_date = models.DateField()
+    introduction    = models.CharField(max_length=200)
+    budget_plan     = models.CharField(max_length=200)
+    organizations   = models.ManyToManyField("Organization", through="ProjectOrganization", related_name="project")
+
+    class Meta:
+        db_table = "projects"
+
+class Category(models.Model):
+    project   = models.ForeignKey(Project, on_delete=models.CASCADE)
+    name      = models.CharField(max_length=50)
+    image_url = models.URLField(max_length=150)
```

Comment:
category와 project의 관계를 다시 생각해 보고 foreignkey가 어디에 위치해야 하는지 생각해 보세요.

```suggestion
class Category(models.Model):
    name      = models.CharField(max_length=50)
    image_url = models.URLField(max_length=150)
```